### ISSUE TITLE
Authentication issue with NetSuite

### ISSUE THREAD

**Issue by maheshbonagiri** [oracle/netsuite-suitecloud-sdk#862]
### Select the SuiteCloud tool where you encountered the issue

SuiteCloud Extension for VSCode

### Version of the SuiteCloud Tool

2.0.1

### Operating system

MacOs

### Node Version / VSCode Version

v18.15.0

### JAVA Version

17.0.6

### Bug description

I have connected with my client netsuite account recently and it is working fine till yesterday and everyday it is asking for authentication for starting of the day. Today i tried to authenticate it again it is showing an error that enable the Oauth 2.0 and SDF, but it is working before and another systems and other users. I have checked the enable features in netsuite it is already enabled. somehow it is showing error idk how to resolve this. Can you please help me out on this issue?

### Steps To Reproduce

setup account steps

### Actual Output

Server error: Verify the arguments of the command that the account has following features enabled: Oauth 2.0 and Suitecloud  Development Framework.

### Expected Output

It is should be connected with netsuite.

**Comment by alisyed-19** [comment]
@maheshbonagiri Do you have multiple accounts assigned to you?
You are right, this error should not be displayed!
Is it possible to meet on a zoom call and share your screen?

**Comment by maheshbonagiri** [comment]
Yes we can connect. Please share the meeting invite to maheshbonagiri058@gmail.com

**Comment by nsconsultant8943** [comment]
I have also experienced this issue. Not sure what is happening. Is there a fix for this that we can perform ourselves? 

Thanks,

**Comment by maheshbonagiri** [comment]
This issue because of the Zscalar. So I have connected with office IT team and they have add the URL Exception policy in the Zscalar and resolved.

**Comment by manuel-morales-bc** [comment]
Hey @maheshbonagiri  we have the same issue, same steps taken and still experiencing it, what was the URL you needed to add to exceptions?
